packages/synapse-react-client/src/components/FeaturedDataTabs/FacetPlotsCardGrid.tsx

```diff
@@ -30,31 +30,14 @@ export const FacetPlotsCardGridContainer: StyledComponent<BoxProps> = styled(
      */
     display: 'grid',
     gridTemplateRows: '[title] auto [plot0] auto',
-    gridTemplateColumns: '[col1] 32% [col2] 32% [col3] 32%',
+    justifyContent: 'center',
+    gridTemplateColumns: 'repeat(auto-fit, minmax(100px, 1fr))',
```

Comment:
Update layout css to support the number of cards other than 3

Reply:
I am so impressed by CSS grids every time I learn about a new (to me) feature